CMake/cmake_modules/ExternalProject.cmake

```diff
@@ -156,6 +156,10 @@
 # (To distribute this file outside of CMake, substitute the full
 #  License text for the above reference.)
 
+# CMake policy usages
+cmake_policy(PUSH)
+cmake_policy(SET CMP0054 NEW)
```

Comment:
This policy was introduced in 3.1.  Probably should add `if(POLICY CMP0054)` to support older cmake's.